### PR TITLE
fix(inputs.lvm): Allow to specify path to binaries in config file

### PR DESCRIPTION
**Pull request by dynek**
- [x] Updated associated README.md.
- [x] Wrote appropriate unit tests
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

Fixes [this problem](https://github.com/influxdata/telegraf/pull/9771#issuecomment-1129038389) mentioned by @rafaelreis-r to @powersj.

I basically followed what is done in other plugins when it comes to make it possible for user to specify path to binary files.
In this specific situation, I need to be able to modify path for `lvs`, `vgs` and `lvs` as they are located in `/sbin/` on my Linux system (Debian Bullseye) rather than `/usr/sbin/` like it is currently hard-coded.

### CODE REVIEW

File: plugins/inputs/lvm/sample.conf [hunk 1]
```diff
@@ -2,3 +2,12 @@
...
   use_sudo = false
+
+  ## The default location of the pvs binary can be overridden with:
+  pvs_binary = "/usr/sbin/pvs"
+
+  ## The default location of the vgs binary can be overridden with:
+  vgs_binary = "/usr/sbin/vgs"
+
+  ## The default location of the lvs binary can be overridden with:
+  lvs_binary = "/usr/sbin/lvs"
```
Comment (srebhan) [inline plugins/inputs/lvm/sample.conf:13]: Can you please comment those options as those should be the defaults!?

File: plugins/inputs/lvm/lvm.go [hunk 2]
```diff
@@ -284,6 +284,11 @@ type lvsReport struct {
 
 func init() {
 	inputs.Add("lvm", func() telegraf.Input {
-		return &LVM{}
+		return &LVM{
+			UseSudo:   false,
```
Comment (srebhan) [inline plugins/inputs/lvm/lvm.go:288]: Please remove this line as this is the Go default. Same in the test cases.

**Review by srebhan — COMMENTED** [review]
Thanks @dynek for your contribution! Only two small comments from my side. Can you please take a look!?

**Review by Hipska — APPROVED** [review]

File: plugins/inputs/lvm/sample.conf (hunk 1 above)
Comment (dynek) [reply plugins/inputs/lvm/sample.conf]: Done in commit https://github.com/influxdata/telegraf/pull/12725/commits/a24a4faaf86e7f5060d50d4ff2e79f74fecee360

File: plugins/inputs/lvm/lvm.go (hunk 2 above)
Comment (dynek) [reply plugins/inputs/lvm/lvm.go]: Done in commit https://github.com/influxdata/telegraf/pull/12725/commits/a24a4faaf86e7f5060d50d4ff2e79f74fecee360

**Review by Hipska — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Looks good to me. Thanks for your contribution @dynek!

**Review by powersj — APPROVED** [review]

**Pull request merged**